sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -368,6 +389,16 @@ export interface PolicySubjectProperties {
     subjectAlternativeNames: SubjectAlternativeNames;
 }
 
+// @public
+export interface PollerLikeWithPublicState<TPublic, TState, TResult> extends PollerLike<TState, TResult> {
+    getPublicState(): TPublic;
```

Comment:
Did we want to consider calling this `getProperties()` to align with .NET?

Reply:
The idea that "state" has "properties" doesn't seem to show why there's this distinction. I can change it if that's the only way to make this better, but I think we somehow should express why we can't have these properties inside the same state properties.

Reply:
I guess `publicState` only is one letter longer than `properties`, it just looks a lot longer to me for some reason

Reply:
Again I'm not a fan of the word "Public" here. We wouldn't ever export a method called getPrivateState so public seems superfluous.

Reply:
Maybe we're going about this backwards and `getOperationState()` should return `PollOperationState<TResult, TCustomProperties>` that has both the standard operation state things plus a property that contains whatever additional state/properties ?